recipes/torchaudio/recipe.yaml

```diff
@@ -0,0 +1,164 @@
+context:
+  name: torchaudio
+  version: "2.4.1"
+  build_number: 0
+  cuda_build_string: cuda_${{ cuda_version | version_to_buildstring }}
+  string_prefix: ${{ cuda_build_string if cuda else "cpu_" }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/pytorch/audio/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: a6b7e1b606ae353b9c4db1c8e754263f53f0457e9ab538e9669c0af0c46a0d5c
+    patches:
+      - patches/0001-point-to-correct-prefix.patch
+
+build:
+  number: ${{ build_number}}
+  skip:
+    - win
+  string: ${{ string_prefix }}py${{ python | version_to_buildstring }}h${{ hash }}_${{ build_number }}
+  variant:
+    use_keys:
+      # use cuda from the variant config, e.g. to build multiple CUDA variants
+      - cuda
+    # this will down-prioritize the cuda variant versus other variants of the package
+    down_prioritize_variant: ${{ 1 if cuda else 0 }}
```

Comment:
I don't think `cuda` is actually set, and the whole cuda logic is probably wrong here.

Revisit once https://github.com/conda-forge/xformers-feedstock/pull/33 has been merged and use it as a cuda template, maybe.